contrib/cf-remote/cf_remote/spawn.py

```diff
@@ -367,7 +367,10 @@ def spawn_vms(vm_requests, creds, region, key_pair=None, security_groups=None,
 
 
 def destroy_vms(vms):
-    with Pool(len(vms)) as pool:
+    n = len(vms)
+    if (n == 0):
+        return
```

Comment:
I'd do `if not vms: return`. It's more generic.

Reply:
Very well. Done.